index.js

```diff
@@ -29,25 +29,16 @@ app.get('/', (req, res) => {
   res.sendFile(__dirname + '/public/index.html')
 })
 
-const urlHash = () => {
-  return Math.random().toString(36).substring(7)
-}
-
-const urlGen = (req) => {
-  return `${req.protocol}://${req.get('host')}${req.originalUrl}`
-}
-
-const liveUrlGen = (req) => {
-  return `${req.protocol}://${req.get('host')}/live_poll`
-}
-
```

Comment:
Yea, I think I like this approach better. That said, I would break them out into one or more separate files.